test/parallel/proffOfStake.3rdPartyRewards.activation.test.ts

```diff
@@ -38,32 +34,37 @@ describe("Proof of stake tests", () => {
     } catch (e) {
       await initApi();
     }
-
-    keyring = new Keyring({ type: "sr25519" });
-    sudo = new User(keyring, getEnvironmentRequiredVars().sudo);
+    sudo = getSudoUser();
     [testUser1, testUser2, testUser3] = setupUsers();
-    [newToken, newToken2, newToken3] = await Assets.setupUserWithCurrencies(
-      sudo,
-      [Assets.DEFAULT_AMOUNT, Assets.DEFAULT_AMOUNT, Assets.DEFAULT_AMOUNT],
-      sudo,
-      true,
-    );
```

Comment:
I think that we should not add more tokens, unless I'm missing something.

Reply:
It was problem on my local node. Change cancelled